### DISCUSSION TITLE
Source code in Ascii Doc is not displayed correctly

### DISCUSSION THREAD

**Discussion by adebuhr** [requarks/wiki#6130] [Error / Bug Report] [answered]
I have tried to use the AsciiDoc editor.

Unfortunately the display of the source code does not work correctly.

Attached the code and the representation

Code:
![image](https://user-images.githubusercontent.com/13552148/217210814-8d5c9dfa-4ff2-4888-b344-02d58ff574bf.png)

Rendering:
![image](https://user-images.githubusercontent.com/13552148/217210909-b9e4b1f4-0e3c-4dac-b279-05c36fd2206d.png)

Is this rendering not supported yet?

**Comment by NGPixel** [comment, marked as answer]
Keep in mind that asciidoc rendering was added very recently and is not yet fully implemented. Many asciidoc features, such as code rendering, are not yet working.

Support will be improved in future updates. I recommend using the Markdown editor if you need to use source code blocks in the meantime.

**Reply by gcleaves** [reply]
Hi. Are the limitations documented anywhere?
